src/app/styles/components/_preview-card.scss

```diff
@@ -55,6 +56,7 @@
         flex-flow: column nowrap;
         align-items: stretch;
         justify-content: space-between;
+        margin-bottom: 5rem;
```

Comment:
Yeah, knew about the unit mixing, but didn't know that about calc. Can give that a shot.